esmvaltool/recipes/ipccwg1ar5ch9/recipe_flato13ipcc_figures_938_941_cmip6.yml

```
@@ -322,13 +322,13 @@ diagnostics:
     variables:
       pr:
         preprocessor: clima_nomask_pr
-        reference_dataset: GPCP-SG
+        reference_dataset: GPCP-V2.2
         mip: Amon
         ensemble: r1i1p1
         start_year: 1980
         end_year: 2005
     additional_datasets:
-      - {dataset: GPCP-SG, project: OBS, project: obs4MIPs, level: L3, version: v2.2, tier: 1}
+      - {dataset: GPCP-V2.2, project: OBS, level: L3, tier: 1}
```

```suggestion
      - {dataset: GPCP-V2.2, project: obs4MIPs, tier: 1}
```
project should be `obs4MIPs` (looks like the project was mistakenly defined twice in this recipe) and `level` is no longer used in the DRS (see https://github.com/ESMValGroup/ESMValCore/blob/6c2e383222659a74f6ccd1e4c2fa20d39c220c73/esmvalcore/config-developer.yml#L105-L115) as this is one of the facets from the pre-2017 obs4MIPs specification (see #2974).